components/areas/AreasForm.js

```diff
@@ -16,8 +16,8 @@ import Input from 'components/form/Input';
 import UploadArea from 'components/areas/UploadArea';
 
 // Services
-import AreasService from 'services/AreasService';
-import UserService from 'services/user';
+import { fetchCountries, createGeostore } from 'services/areas';
```

Comment:
https://github.com/resource-watch/resource-watch/pull/1116/files#r324716188 <- same. The services have a unique purpose.